### ISSUE TITLE
Fails under windows with non latin characters

### ISSUE THREAD

**Issue by patl12345** [mirumee/google-i18n-address#18]
I feel like i'm being 'that guy' but when it tries to load cn.json (and possibly others, I haven't tested) it fails because windows defaults to cp1252 rather than utf-8, explicitly specifying the encoding when the json file is opened would fix it though.
```
diff --git a/i18naddress/__init__.py b/i18naddress/__init__.py
index 0d90d9e..5df995a 100644
--- a/i18naddress/__init__.py
+++ b/i18naddress/__init__.py
@@ -32,7 +32,7 @@ def load_validation_data(country_code='all'):
     if not os.path.exists(path):
         raise ValueError(
             '%r is not a valid country code' % (country_code,))
-    with open(path) as data:
+    with open(path, encoding='utf-8') as data:
         return json.load(data)
```

**Comment by mociepka** [comment]
It looks like python 3 feature. If you don't tell interpreter to use `utf-8` it pickups `ASCII`. On Linux you can set `LC_ALL` to for instance `us_US.UTF-8` with that env interpreter uses `utf-8`.

I don't knew what environment should be set if you figure it out please comment.

**Comment by patrys** [comment]
Fixed in `2.0.1`. 👍
